src/mono/wasm/runtime/memory.ts

```diff
@@ -114,3 +136,7 @@ export function getF32(offset: _MemOffset): number {
 export function getF64(offset: _MemOffset): number {
     return Module.HEAPF64[<any>offset >>> 3];
 }
+
+export function afterUpdateGlobalBufferAndViews(buffer: Buffer): void {
+    HEAPI64 = new BigInt64Array(buffer);
```

Comment:
we should probe whether this type is available in the browser and generate a meaningful error if it isn't

Reply:
done